src/screens/ToDoList/ToDoList.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { PureComponent } from 'react';
+import ButtonComponent from '../../shared/components/Input/Button/Button';
+import InputComponent from '../../shared/components/Input/Input/Input';
+import ToDoListItemComponent from './ToDoListItem/ToDoListItem';
+import { generateId } from '../../shared/utils/idGenerator';
+import { saveItemInLocalStorage, getItemFromLocalStorage } from '../../shared/utils/LocalStorage';
+import './styles.scss';
+
+class ToDoListComponent extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+      newItem: '',
+
+    }
+
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onAddNewItemClick = this.onAddNewItemClick.bind(this);
+    this.markCompleted = this.markCompleted.bind(this);
+    this.deleteItem = this.deleteItem.bind(this);
+    this.deleteAllItems = this.deleteAllItems.bind(this);
+  }
+
+  componentDidMount() {
+    const tasks = getItemFromLocalStorage('items') || [];
+    this.setState({items: tasks})
```

Comment:
Missing semicolon. Object should have additional spaces - `{ items: tasks }`

Reply:
Done!